stacker/blueprints/rds/base.py

```diff
@@ -0,0 +1,336 @@
+from troposphere import (
+    Ref, ec2, Output, GetAtt, Not, Equals, Condition, And, Join, If
+)
+from troposphere.rds import DBInstance, DBSubnetGroup
+from troposphere.route53 import RecordSetType
+
+from ..base import Blueprint
+
+RDS_ENGINES = ["MySQL", "oracle-se1", "oracle-se", "oracle-ee", "sqlserver-ee",
+               "sqlserver-se", "sqlserver-ex", "sqlserver-web", "postgres"]
+
+# Resource name constants
+SUBNET_GROUP = "RDSSubnetGroup"
+SECURITY_GROUP = "RDSSecurityGroup"
+DBINSTANCE_NO_IOPS = "RDSDBInstanceNoIops"
+DBINSTANCE_WITH_IOPS = "RDSDBInstanceWithIops"
+DNS_RECORD = "DBInstanceDnsRecord"
+
+
+def common_parameters():
+    """ Return common parameters for all RDS stacks.
+
+    Returns:
+        dict: A dictionary of parameter definitions.
+    """
+
+    parameters = {
```

Comment:
we could just leave that up to private stacks but make it so we can pass those parameters to the db instance troposphere instance